PXWeb/Code/Views/ViewSerializerCreator.cs

```diff
@@ -24,8 +24,6 @@ public static IViewSerializer GetSerializer(string type)
                     return new TableViewSerializer();
                 case PxUrl.VIEW_CHART_IDENTIFIER :
                     return new ChartViewSerializer();
-                case PxUrl.VIEW_FOOTNOTES_IDENTIFIER :
-                    return new FootnoteViewSerializer();
```

Comment:
Maybe we should just return the TableViewSerializer instead since the footnotes are located there and we might have saved queries going to the footnote pages.

Reply:
I agree with woth you and changed to return TableViewSerializer()